app/modules/asset_groups/parameters.py

```diff
@@ -48,3 +48,36 @@ def replace(cls, obj, field, value, state):
     @classmethod
     def remove(cls, obj, field, value, state):
         raise NotImplementedError()
+
+
+class PatchAssetGroupSightingDetailsParameters(PatchJSONParameters):
+    # pylint: disable=abstract-method,missing-docstring
+    OPERATION_CHOICES = (PatchJSONParameters.OP_REPLACE, PatchJSONParameters.OP_ADD)
+    VALID_FIELDS = ['config']
+    PATH_CHOICES = tuple('/%s' % field for field in VALID_FIELDS)
+
+    @classmethod
+    def add(cls, obj, field, value, state):
+        # Add and replace are the same operation so reuse the one method
+        return cls.replace(obj, field, value, state)
+
+    @classmethod
+    def replace(cls, obj, field, value, state):
+
+        ret_val = False
+        # Permissions for all fields are the same so have one check
+        if (
+            current_user.is_privileged
+            or current_user.is_admin
+            or current_user.is_researcher
+        ):
```

Comment:
Good question. Not exactly sure. Copied from elsewhere which now you mention it may also be wrong. 
If you want to merge this in to get the next step of the IA pipeline in I'll look at this later

Reply:
I fixed this one, It shouldn't have been there